queries/tiger/highlights.scm

```diff
@@ -23,6 +23,7 @@
 [
  "function"
  "primitive"
+ "method"
```

Comment:
`method` is for declaring new methods on classes, so it should be `@method` according to `CONTRIBUTING.md`.

Reply:
`@method` is for the method name.
`@keyword.function` is for the keyword that defines it.
There's no `@keyword.method` distinction.

Reply:
Alright, makes sense to me.